scripts/ofekQuery.js

```diff
@@ -0,0 +1,123 @@
+function $(query){
+	return new queryResults(query);
+
+}
+
+var queryResults = function(query){
```

Comment:
OfekQuery is the object name, not queryResults. AND! you are calling queryResult with new - meaning its a class.
Use capital letters - OfekQuery